Library/Formula/grails.rb

```diff
@@ -17,7 +17,9 @@ def caveats; <<-EOS.undent
     EOS
   end
 
-  test do
-    assert_match /Application created/, shell_output("#{bin}/grails create-app testApp")
-  end
+  +  test do
```

Comment:
You seem to have applied a diff file down here, rather than the diff itself. You've got `+  test do` rather than `test do` and so on.